python-checks/src/test/resources/checks/robustCipherAlgorithm.py

```diff
@@ -98,3 +114,19 @@ def pycryptoCompliant():
   from Crypto.Cipher import *
   aes1 = AES.new('This is a key123', AES.MODE_CBC, 'This is an IV456') # Compliant
 
+def pysslCompliant():
+  import ssl
+  ciphers = 'ECDHE-RSA-AES256-GCM-SHA384'
+  ctx = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+  ctx.set_ciphers(ciphers)
+  ciphers2 = ''
+  ctx.set_ciphers(ciphers2)
+  ciphers3 = 'ECDHE:RSA:AES256:GCM:SHA384'
+  ctx.set_ciphers(ciphers3)
+
+  ciphers4 = 1
+  ctx.set_ciphers(ciphers4)
+  ctx.set_ciphers(ciphers5)
```

Comment:
Here if you are testing with the `None` value it would be better to explicitly state it.

Reply:
I made the change in 724de8800a0cff9aef4f33ca3d0e813f80acfb00.